package.json

```
@@ -240,73 +240,74 @@
 		"worker-farm": "1.7.0"
 	},
 	"scripts": {
-		"analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
-		"api-docs:blocks": "node ./bin/api-docs/gen-block-lib-list.js",
-		"api-docs:ref": "node ./bin/api-docs/update-api-docs.js",
-		"api-docs:theme-ref": "node ./bin/api-docs/gen-theme-reference.js",
-		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
-		"clean:package-types": "tsc --build --clean",
+		"build": "npm run build:packages && wp-scripts build",
+		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "npm run build:package-types && node ./bin/packages/build.js",
-		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
 		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
-		"build": "npm run build:packages && wp-scripts build",
-		"changelog": "node ./bin/plugin/cli.js changelog",
-		"cherry-pick": "node ./bin/cherry-pick.mjs",
-		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
-		"precheck-local-changes": "npm run docs:build",
-		"check-local-changes": "node ./bin/check-local-changes.js",
+		"clean:package-types": "tsc --build --clean",
+		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
 		"dev": "npm run build:packages && concurrently \"wp-scripts start\" \"npm run dev:packages\"",
 		"dev:packages": "concurrently \"node ./bin/packages/watch.js\" \"tsc --build --watch\"",
 		"distclean": "rimraf node_modules packages/*/node_modules",
+		"docs:blocks": "node ./bin/api-docs/gen-block-lib-list.js",
+		"docs:build": "npm-run-all docs:gen docs:blocks docs:ref docs:theme-ref",
 		"docs:gen": "node ./docs/tool/index.js",
-		"docs:build": "npm-run-all docs:gen api-docs:*",
+		"docs:ref": "node ./bin/api-docs/update-api-docs.js",
```

Maybe let's make it explicit it's related to public APIs:
```suggestion
		"docs:api-ref": "node ./bin/api-docs/update-api-docs.js",
```

---

Fixed with 6c7ae39